content/blog/eu_digital_sovereignty.md

```diff
@@ -0,0 +1,83 @@
+# EU Digital Sovereignty at Risk
+
+## Introduction
+
```

Comment:
I lack an intro on why sovereignty is an issue, and how we came up to this point.
Why did we move massively to US silicon valley clouds?